src/index.tsx

```diff
@@ -1,53 +1,70 @@
-import React, { useEffect, useRef, useState } from 'react'
-import PropTypes from 'prop-types'
+// eslint-disable-next-line
+import React, { useEffect, useRef, useState, MouseEvent, RefObject, ReactNode } from 'react'
 
 import { roundNum } from './utils'
 import { applyDragForce, applyBoundForce } from './force'
 import getBoundaries from './getBoundaries'
+// TypeScript does not like using the import statement for non js/ts files
+const styles = require('./styles.css')
```

Comment:
Actually you can import css files, you just need to create a `typings.d.ts` in src with following content:

    /**
     * Default CSS definition for typescript,
     * will be overridden with file-specific definitions by rollup
     */
    declare module "*.css" {
      const content: { [className: string]: string };
      export default content;
    }

Reply:
this didn't work for me. I'm leaving the rollup config modifications for now. test is now passing

Reply:
yeah no need to actually do this. I am just telling its possible.